tests/__init__.py

```diff
@@ -99,6 +99,17 @@ class FoobarParams:
     params: FoobarParams = FoobarParams()
 
 
+@dataclass
+class NestedInterpolationToMissing:
+    name: str = MISSING
+
+    @dataclass
+    class BazParams:
+        baz: str = "${..name}"
+
+    subcfg: BazParams = BazParams()
+
+
```

Comment:
You can test this without a full fledged dataclass.
See test_dict_values_with_missing_value.

Reply:
That works for the `to_container` tests, but I think a dataclass is necessary for testing `OmegaConf.to_object`/`DictConfig._to_object`.

Reply:
It is. Resolving.